datastore/ndb/modeling/tests/test_contact_with_group_models.py

```diff
@@ -19,9 +19,10 @@
 
 from google.appengine.ext import ndb
 
-import contact_with_group_models as models
 import test_base
 
+from .. import contact_with_group_models as models
```

Comment:
We need relative imports because the top level dir is not a package right? Would it be worth making the whole repo a package so we could have absolute imports?